src/coreclr/vm/arm64/crthelpers.S

```diff
@@ -12,7 +12,7 @@
 LEAF_ENTRY JIT_MemSet, _TEXT
     cbz x2, LOCAL_LABEL(JIT_MemSet_ret)
 
-    strb w1, [x0]
+    ldrb wzr, [x0]
```

Comment:
this is now checking that the memory is readable vs. writable, but achieves the same purpose?

Reply:
Yes, the purpose of this code is to really only check for null.